examples/influxdb-cluster-simple/README.md

```diff
@@ -14,9 +14,7 @@ You will need to create an [Amazon Machine Image (AMI)](http://docs.aws.amazon.c
 that has InfluxDB installed, which you can do using the [influxdb-ami 
 example](https://github.com/gruntwork-io/terraform-aws-influx/tree/master/examples/influxdb-ami)). 
 
-To see an example of InfluxDB deployed across separate clusters, see the [influxdb-multi-cluster
-example](https://github.com/gruntwork-io/terraform-aws-influx/blob/master/examples/influxdb-multi-cluster). For
-more info on how the InfluxDB cluster works, check out the 
```

Comment:
Should link to https://github.com/gruntwork-io/terraform-aws-influx/tree/master/examples/tick-multi-cluster